src/tools/build-manifest/src/main.rs

```diff
@@ -489,6 +502,7 @@ impl Builder {
             manifest.renames.insert(from.to_owned(), Rename { to: to.to_owned() })
         };
         rename("rls", "rls-preview");
+        rename("rust-analyzer", "rust-analyzer-preview");
```

Comment:
Either just add the component as `rust-analyzer` and not do the rename thing, or else don't include the rename for now perhaps?  If this is a nightly-only component then the `-preview` is unnecessary surely?

cc @pietroalbini